sdk/formrecognizer/Azure.AI.FormRecognizer/tests/samples/SampleSnippets.cs

```diff
@@ -0,0 +1,85 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Threading.Tasks;
+using Azure.AI.FormRecognizer.DocumentAnalysis.Tests;
+using Azure.Core.TestFramework;
+using NUnit.Framework;
+
+namespace Azure.AI.FormRecognizer.DocumentAnalysis.Samples
+{
+    /// <summary>
+    /// Samples that are used in the associated README.md file.
+    /// </summary>
+    public partial class Snippets : SamplesBase<DocumentAnalysisTestEnvironment>
+    {
+        [Test]
+        public void CreateDocumentAnalysisClient()
```

Comment:
Should we update the section in the README to use the new snippets (create the clients + bad request)? Currently it still points to the V2 ones.

Reply:
Also, do you think we should include the `V3` suffix to the snippets in `V3/SampleSnippets.cs`?

Reply:
I want to avoid working on Readme yet and just leave it to its own PR.

Reply:
> I have only done it for the ones that might collide because names are the same

Fair enough. Just wanted to make sure you didn't miss those by accident.

Reply:
Thank you!